### PR TITLE
Add new permisssions

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Fixes https://issues.redhat.com/browse/AAH-956

Depends on https://github.com/ansible/galaxy_ng/pull/1003 (merged)

TODO:

- [x] Delete containers: pulp_container.delete_containerrepository

Screenshot 2021-10-05 at 11 13 29

- [x] Delete collections: ansible.delete_collection

Screenshot 2021-10-05 at 11 16 24

- [x] Delete namespace: galaxy.delete_namespace

Screenshot 2021-09-29 at 13 53 57

- [x] Delete registry: galaxy.delete_containerregistryremote

Screenshot 2021-09-29 at 13 00 00

- [x] Create Registry: galaxy.add_containerregistryremote
Screenshot 2021-09-29 at 12 50 11

- [x] Change registry: galaxy.change_containerregistryremote

Screenshot 2021-09-29 at 12 59 27

### CODE REVIEW

File: src/containers/execution-environment/registry-list.tsx [hunk 1]
```diff
@@ -116,7 +116,11 @@ class ExecutionEnvironmentRegistryList extends React.Component<
     const noData =
       items.length === 0 && !filterIsSet(params, ['name__icontains']);
 
-    const addButton = (
+    if (this.context.user.is_anonymous) {
+      return <EmptyStateUnauthorized />;
+    }
+
+    const addButton = this.context.user.model_permissions.add_remote ? (
```
Comment (himdel) [inline src/containers/execution-environment/registry-list.tsx:123]: do we need to add `add_remote` to `src/api/response-types/user.ts`, along with the rest?
(also, any idea why none of those 3 are in src/constants?)

File: src/containers/execution-environment/registry-list.tsx [hunk 2]
```diff
@@ -415,7 +419,7 @@ class ExecutionEnvironmentRegistryList extends React.Component<
                   <Trans>Index execution environments</Trans>
                 </DropdownItem>
               </Tooltip>,
-            ]}
+            ].filter(Boolean)}
```
Comment (himdel) [inline src/containers/execution-environment/registry-list.tsx:422]: :+1: nice

But also, all the items can be filtered out now, should we move this into an array, and only show the dropdown depending on `length`?

Comment (ZitaNemeckova) [reply src/containers/execution-environment/registry-list.tsx]: Fixed

File: src/containers/execution-environment/registry-list.tsx [hunk 3]
```diff
@@ -354,6 +354,52 @@ class ExecutionEnvironmentRegistryList extends React.Component<
   }
 
   private renderTableRow(item: any, index: number) {
+    const dropdownItems = [
+      this.context.user.model_permissions.change_remote && (
```
Comment (himdel) [inline src/containers/execution-environment/registry-list.tsx:358]: the `change_remote` permission is also used for the remote repositories (of collections), should it be the same one?

**Comment by himdel** [comment]
The changes look good, but I don't think `add_remote`, `change_remote` and `delete_remote` are the right permissions for container registries, given that we also have `add_containerregistry`, `change_containerregistry` and `delete_containerregistry` (from https://github.com/ansible/galaxy_ng/pull/1003)

**Review by himdel — APPROVED** [review]
LGTM,
verified this stops an unpriviledged user from adding/changing/removing these registries :+1: 
and removing container repositories :+1:

**Pull request merged**